crates/xilem_html/src/view.rs

```diff
@@ -213,6 +211,94 @@ where
     }
 }
 
+impl<T, A, V1, V2> ViewSequence<T, A> for Either<V1, V2>
+where
+    V1: ViewSequence<T, A>,
+    V2: ViewSequence<T, A>,
+{
+    type State = Either<V1::State, V2::State>;
+
+    fn build(&self, cx: &mut Cx, elements: &mut Vec<Pod>) -> Self::State {
+        match self {
+            Either::Left(view) => Either::Left(view.build(cx, elements)),
+            Either::Right(view) => Either::Right(view.build(cx, elements)),
+        }
+    }
+
+    fn rebuild(
+        &self,
+        cx: &mut Cx,
+        prev: &Self,
+        state: &mut Self::State,
+        element: &mut xilem_core::VecSplice<Pod>,
+    ) -> ChangeFlags {
+        match (prev, self) {
+            (Either::Left(_), Either::Right(view)) => {
```

Comment:
```suggestion
            (Either::Left(_), Either::Right(view_sequence)) => {
```

As well for the other `view`s in the `impl`.